### ISSUE TITLE
Chapter 31 - Remove `data-gtm` tag from iframe on each video change

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/va.gov-team#20394]
## Description 
Contextual issue - https://github.com/department-of-veterans-affairs/va.gov-team/issues/15757

The analytics team recently discovered that GA was only tracking initial video load on the chapter 31 orientation flow, instead of tracking all videos. As mentioned in the above Github issue, the possible reason for this is the inclusion of a `data-gtm-yt-inspected-XXXXXXX_XXX` attribute. The hypothesis is that removing this each time a new video is loaded will enable GA to once again track video loads reliably. 

## Considerations
- Users can navigate forward and backwards between steps, so any attribute removal mechanism will have to accommodate both scenarios
- It isn't 100% certain this will solve the issue, so once a solution is in place, the eBenefits team and Analytics team will need to collaborate on testing this before it is merged into master
- Add `enablejsapi=1` as a query parameter to the iframe URL - [more here](https://developers.google.com/youtube/player_parameters#enablejsapi)
- [Convo with Jon](https://dsva.slack.com/archives/CLY6Q69RV/p1614023548002700)

## Tasks
- [x] Investigate and implement a potential solution 
- [x] add `enablejsapi=1` as a query parameter
- [x] Test before merging

## Acceptance criteria
- [x] potential solution has been implemented locally
- [x] potential solution has been tested locally before merge
